templates/BeamService/Dockerfile

```diff
@@ -1,5 +1,15 @@
-﻿# use the dotnet SDK as a build stage
-FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine AS build-env
+﻿ARG BUILD_MODE=beamable_dev
+
+# Processes specific 
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine as build_game_maker
```

Comment:
I am anxious about putting this in the default template for all customer microservices. I understand the value to _us_, but the value to our customers feels very low, and the clutter it costs in the dockerfile feels bad. 

I'm fine with this for now, but I wonder if there is a world where we replace this with more generated dockerfile nonsense. Like, 

```Dockerfile
# <BEAM-CLI-PROCESS> this line signals the start of custom logic injected by the CLI based upon the --build-mode property.

# </BEAM-CLI-PROCESS>
```